README.md

```diff
@@ -91,12 +94,13 @@ head.title
 head.base
 head.meta
 head.link
+head.script
 ```
 
-`head` contains four properties, `title`, `base`, `meta`, and `link`:
+`head` contains five properties, `title`, `base`, `meta`, `link`, `script`:
 
 - `title` returns a string.
```

Comment:
Should we return a `<title>` component instead? This would make `title` return the same as the other supported tags.